test/runnable/gdb1.d

```diff
@@ -8,7 +8,7 @@ r ARG1 ARG2
 echo RESULT=
 p args
 ---
-GDB_MATCH: RESULT=.*ARG1.*ARG2
```

Comment:
My bad. Since the array contains the full path as zero-eth element gdb decides to split the content over many lines, making the regex fail. I've never bothered to submit a fixed test and I guess this is the right moment to do so.